tests/testthat/fun_dec/line_break_fun_dec-out.R

```diff
@@ -12,6 +12,6 @@ a <- function(x, #
 
 a <- function(x, #
               y #
-            ) { # FIXME: Move to the same indention level as a
+) {
```

Comment:
Do we need to indent this closing paren?

Reply:
I think it makes sense to not indent it (this is also what happens in RStudio) because a closing brace unindents itself. Alternative to leave it as it was before is moving it to the same level as `)`, which I think looks worse plus it's more complicated 😩.

Reply:
Also, this is only relevant for cases where the closing brace cannot go directly after the last formal, because it is blocked by a comment. Otherwise the brace moves up. Debatable whether we want that or not, but I think it's a detail.